charts/fluentbit-operator/templates/fluentbit-operator-deployment.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fluentbit-operator
+  labels:
+    app.kubernetes.io/component: operator
+    app.kubernetes.io/name: fluentbit-operator
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: operator
+      app.kubernetes.io/name: fluentbit-operator
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: operator
+        app.kubernetes.io/name: fluentbit-operator
+    spec:
+      {{- if eq .Values.containerRuntime "docker" }}
+      volumes:
+      - name: env
+        emptyDir: {}
+      - name: dockersock
+        hostPath:
+          path: /var/run/docker.sock
+      initContainers:
+      - name: setenv
+        image: {{ .Values.operator.initcontainer.image }}:{{ .Values.operator.initcontainer.tag }}
+        command:
+        - /bin/sh
+        - '-c'
+        - set -ex;
+          echo DOCKER_ROOT_DIR=$(docker info -f '{{`{{ .DockerRootDir }}`}}') > /fluentbit-operator/fluent-bit.env
+        volumeMounts:
+        - name: env
+          mountPath: /fluentbit-operator
+        - name: dockersock
+          readOnly: true
+          mountPath: /var/run/docker.sock
+      {{- else if eq .Values.containerRuntime "containerd" }}
+      volumes:
+      - name: env
+        emptyDir: {}
+      - name: containerdsock
+        hostPath:
+          path: /var/run/containerd/containerd.sock
+      - name: crictl
+        hostPath:
+          path: /usr/local/bin/crictl
+      initContainers:
+      - name: setenv
+        image: {{ .Values.operator.initcontainer.image }}:{{ .Values.operator.initcontainer.tag }}
+        command:
+        - /bin/sh
+        - '-c'
+        - set -ex;
+          echo DOCKER_ROOT_DIR=$(crictl info --output go-template --template '{{`{{ .config.containerdRootDir }}`}}') > /fluentbit-operator/fluent-bit.env
```

Comment:
Maybe `LOG_ROOT_DIR` or `CONTAINER_ROOT_DIR` is more suitable?

Reply:
Just to match this https://github.com/fluent/fluentbit-operator/blob/master/cmd/manager/main.go#L59

Reply:
I have finished the modification, please review it